app/projects/queries/searchProjects.ts

```diff
@@ -125,10 +126,13 @@ export default resolver.pipe(
         whereString = whereString.replace("?", "'" + where.values[strIdx]?.toString() + "'" || "")
       strIdx++
     }
+    whereString += ` OR pm.profileId == '${session.profileId}'`
```

Comment:
I think this logic should be on line 71 instead of here, because right now we are adding this OR condition to all queries, even when not searching for my projects.